packages/financial-templates-lib/src/price-feed/BasketSpreadPriceFeed.js

```diff
@@ -112,6 +113,23 @@ class BasketSpreadPriceFeed extends PriceFeedInterface {
     return this._getSpreadFromBasketPrices(experimentalPrices, baselinePrices, denominatorPrice);
   }
 
+  debugHistoricalData(time) {
```

Comment:
Just a random idea -- have you thought about returning an error message from the getHistoricalData method itself? That way, we wouldn't need to add a new method to the interface. The logic you have here would just be baked into the normal method. We could even say the second element is optional, so if you don't provide it, you're just failing to provide the caller with info about the error. Something like:
```js
return [price, errorString];
```

`errorString` could be null in the success case and price could be null in the error case?

An even more nuclear and invasive option is to begin allowing these price feeds to throw errors and relying on the calling code to catch those errors?

What do you think?

Reply:
I think this would create more churn by changing the return interface of `getHistoricalPrice()`